modules/assim.sequential/R/Remote_helpers.R

```diff
@@ -468,11 +468,10 @@ alltocs <-function(fname="tocs.csv") {
         )
         return(dfout)
       }) %>%
-      mutate(ExecutionTimeP = c(min(TimeElapsed), diff(TimeElapsed))) %>%
-      write.table(
+      dplyr::mutate(ExecutionTimeP = c(min(TimeElapsed), diff(TimeElapsed))) %>%
```

Comment:
Does timeElapsed need a data-mask here?
```suggestion
      dplyr::mutate(ExecutionTimeP = c(min(.data$TimeElapsed), diff(.data$TimeElapsed))) %>%
```